test/docker-compose.typecheck-tests.yml

```diff
@@ -0,0 +1,122 @@
+version: "3.8"
+
+# environment variable PWD is assumed to be grapl root directory
+
+services:
+
+  #
+  # Python services
+  #
+
+  typecheck-analyzer-executor:
```

Comment:
We can table this for another later diff, but a thought I had is that

it would be nice if the typecheck were part of a dockerfile RUN command - that way, the result would be cached  and doesn't have to run again - but hey, not a big deal

(and if every python image gets cache-obliterated anyway if a single service in python/ changes, then, hey whatever)

Reply:
Yeah, that sounds good to me. For another diff preferably.